config/jobs/kubernetes/sig-testing/coverage.yaml

```
@@ -145,7 +149,11 @@ periodics:
         result=0
         ( cd hack/tools && GO111MODULE=on go install gotest.tools/gotestsum ) || result=$?
         make test KUBE_COVER=y KUBE_COVER_REPORT_DIR="${ARTIFACTS}" KUBE_TIMEOUT="--timeout=300s" || result=$?
-        grep -v -E 'zz_generated|generated\.pb\.go' "${ARTIFACTS}/combined-coverage.out" > "${ARTIFACTS}/filtered.cov" || result=$?
+        cd ../test-infra/gopherage
+        GO111MODULE=on go build .
+        export JUNIT=$(find ${ARTIFACTS} -type f -iname 'junit*.xml')
+        ./gopherage filter --exclude-path="zz_generated,generated\.pb\.go"  "${ARTIFACTS}/combined-coverage.out" > "${ARTIFACTS}/filtered.cov" || result=$?
+        ./gopherage junit --threshold 0 "${ARTIFACTS}/filtered.cov" > "${JUNIT}" || result=$?
```

If the goal is to ensure there are no junit.xml files other than the one we're generating, I would go about it this way.  As written if there were ever multiple junit.xml files I'm not sure what the `> ${JUNIT}` would do.
```suggestion
        find ${ARTIFACTS} -type f -iname 'junit*.xml' -exec rm {} \;
        ./gopherage filter --exclude-path="zz_generated,generated\.pb\.go"  "${ARTIFACTS}/combined-coverage.out" > "${ARTIFACTS}/filtered.cov" || result=$?
        ./gopherage junit --threshold 0 "${ARTIFACTS}/filtered.cov" > junit-coverage.xml || result=$?
```

---

There should only ever be one but I can make the changes to make it more robust 👍